scenarios/mixed_arc_must_get_agent_activity/src/main.rs

```diff
@@ -0,0 +1,285 @@
+use holochain_types::prelude::ActionHash;
+use holochain_types::prelude::AgentPubKey;
+use holochain_types::prelude::Timestamp;
+use holochain_wind_tunnel_runner::prelude::*;
+use holochain_wind_tunnel_runner::scenario_happ_path;
+use validated_must_get_agent_activity_coordinator::{
+    BatchChainTop, CreateEntriesBatchInput, GetChainTopForBatchInput, SampleEntryInput,
+};
+
+const RECORD_OPEN_CONNECTIONS_PERIOD_MS: i64 = 3_000;
+const BATCH_SIZE: u32 = 10;
+
+#[derive(Debug, Default)]
+pub struct ScenarioValues {
+    write_peer: Option<AgentPubKey>,
+    batch_count: u32,
```

Comment:
For each write behaviour run, a batch of 10 entries get created. This `batch_count` keeps track of how many such batches have been created so far.